docs/troubleshooting.md

```diff
@@ -217,3 +217,81 @@ The following is a simple example of this occurring:
+| net_cep_deltas                                   | Number of endpoint events sent to Sensor.                                                                                            |
+| net_cep_inactive                                 | Accumulated number of endpoints destroyed (closed)                                                                                   |
+| net_known_ip_networks                            | Number of known-networks defined.                                                                                                    |
+| net_known_public_ips                             | Number of known public addresses defined.                                                                                            |
+| process_lineage_counts                           | Every time the lineage info of a process is created (signal emitted)                                                                 |
+| process_lineage_total                            | Total number of ancestors reported                                                                                                   |
+| process_lineage_sqr_total                        | Sum of squared number of ancestors reported                                                                                          |
+| process_lineage_string_total                     | Accumulated size of the lineage process exec file paths                                                                              |
+| process_info_hit                                 | Accessing originator process info of an endpoint with data readily available.                                                        |
+| process_info_miss                                | Accessing originator process info of an endpoint ends-up waiting for Falco to resolve data.                                          |
+| rate_limit_flushing_counts                       | Number of overflows in the rate limiter used to send process signals.                                                                |
+
+
+### Sysdig stats -> rox_collector_events
+
+| Name                                 | Description                                                                                         |
+|--------------------------------------|-----------------------------------------------------------------------------------------------------|
+| kernel                               | number of received kernel events (by the probe)                                                     |
+| drops                                | number of dropped kernel events                                                                     |
+| preemptions                          | Number of preemptions (?)                                                                           |
+| filtered[syscall]                    | Number of events after chisel filtering                                                             |
```

Comment:
Probably worth mentioning that the form `<name>[syscall]` means one have to look for events that correspond to particular syscalls and under those filtered/userspace/chicel.../etc will be found.

Reply:
Added a short sentence, however, I doubt people are going to consume those metrics programmatically without first taking a look at what the dump actually looks like. The documentation is merely there to say what is available and the meaning of it.